package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tslint-microsoft-contrib",
-  "version": "5.2.1",
+  "version": "6.0.0-beta1",
```

Comment:
All mentions above and in title are `6.0.0-beta0`

Reply:
Thinking about this a bit - I doubt there'll be multiple betas, so I'll just switch it to `6.0.0-beta`.